contribute/en/Release-Process.md

```diff
@@ -328,7 +328,8 @@ Please find all the details in the [Changelog](https://github.com/jamulussoftwar
 _Windows users: Please note that in the first days after release SmartScreen will probably display warnings about this release being unknown upon download and/or execution of the installer. Let us know when you do not see this warning anymore and we will update this announcement accordingly._
 
 **[↓ Windows](<!-- direct link to Windows version -->)** (ASIO version), alternative: [↓ JACK version](<!-- direct link to JACK version -->)
-**[↓ macOS](<!-- direct link to macOS SIGNED version -->)** for High Sierra (10.13) to Big Sur (11) and [↓ macOS legacy build](<!-- direct link to macOS legacy version -->) (unsigned) for macOS Sierra (10.12), El Capitan (10.11) or Yosemite (10.10).
+**[↓ macOS (Intel)](<!-- direct link to macOS SIGNED Intel version -->)** for Catalina (10.15) and higher and [↓ macOS legacy build](<!-- direct link to macOS legacy version -->) (unsigned) for macOS Mojave (10.14) down to Yosemite (10.10).
+**[↓ macOS (Apple Silicon)](<!-- direct link to macOS SIGNED ARM version -->)** for Big Sur (11.0) or later (needs ARM chip)
```

Comment:
How do Apple refer to these?  M-series?

Reply:
Not sure. I think it's apple silicon
https://support.apple.com/en-us/HT211814 or Apple Chip

Reply:
Yep seems to be okay.